packages/common/src/operations.graphql

```diff
@@ -591,3 +592,136 @@ mutation upsertOutboundShipment(
     }
   }
 }
+
+mutation upsertInboundShipment(
+  $deleteInboundShipmentLines: [DeleteInboundShipmentLineInput!]
+  $insertInboundShipmentLines: [InsertInboundShipmentLineInput!]
+  $updateInboundShipmentLines: [UpdateInboundShipmentLineInput!]
+  $updateInboundShipments: [UpdateInboundShipmentInput!]
+) {
+  batchInboundShipment(
+    deleteInboundShipmentLines: $deleteInboundShipmentLines
+    insertInboundShipmentLines: $insertInboundShipmentLines
+    updateInboundShipmentLines: $updateInboundShipmentLines
+    updateInboundShipments: $updateInboundShipments
+  ) {
+    __typename
+    updateInboundShipments {
+      __typename
+      id
+    }
+
+    insertInboundShipmentLines {
+      __typename
+      id
+    }
+
+    deleteInboundShipmentLines {
+      __typename
+      id
+    }
+
+    updateInboundShipmentLines {
+      __typename
+      id
+    }
+  }
+}
+
+mutation updateInboundShipment($input: UpdateInboundShipmentInput!) {
+  updateInboundShipment(input: $input) {
+    ... on InvoiceNode {
+      __typename
+      id
+    }
+    ... on NodeError {
+      __typename
+      error {
+        description
+        ... on DatabaseError {
+          __typename
+          description
+          fullError
+        }
+        ... on RecordNotFound {
+          __typename
+          description
+        }
+      }
+    }
+    ... on UpdateInboundShipmentError {
+      __typename
+      error {
+        description
+      }
+    }
+  }
+}
+
+mutation deleteInboundShipments($ids: [DeleteInboundShipmentInput!]) {
```

Comment:
oddly this is an object `{id: string}` (`DeleteInboundShipmentInput`) while deleting outbounds takes a string array

Reply:
hm. would be good to change the inbound version. I think that might have been done first